README.adoc

```diff
@@ -552,6 +552,32 @@ validates_length_of :email, maximum: 100
 validates :email, presence: true, length: { maximum: 100 }
 ----
 
+=== Custom Validation Methods
+
+When naming custom validation methods, adhere to the simple rules:
+
+ - `validate :method_name` reads like a natural statement
+ - the method name explains what it checks
+ - the method is recognizable as a validation method by its name, not a predicate method
+
+[source,ruby]
+----
+# good
+validate :expiration_date_cannot_be_in_the_past
+validate :discount_cannot_be_greater_than_total_value
+validate :ensure_same_topic
+
+# also good - explicit prefix
+validate :validate_birthday_in_past
```

Comment:
Repeating `validate_` in the name seems a bit awkward...

Reply:
Indeed. However, it seems it's a [widespread practice](https://github.com/rubocop-hq/rails-style-guide/issues/223#issuecomment-358597186).
[Here's an explanation](https://github.com/rubocop-hq/rails-style-guide/issues/223#issuecomment-522912340):

> methods are what makes up the API that's going to be reused. If I'm scanning through the method list and see something like name_must_be_snake_case, I don't immediately know what that method does. It could be used for validation but it could also return a boolean that indicates if name must be snake_case or not. If that method were called validate_name_is_snake_case, I'd know at once that it's not something I'd want to reuse for anything other than validation.

I don't feel like to be using this variation myself, but we don't insist on using it and provide the freedom to use other naming options.